draft-ietf-rats-corim.md

```diff
@@ -2648,16 +2648,17 @@ This includes the following aspects:
 - Conducting regular, automated audits and reviews of the system, such as ensuring that users' privileges are correctly configured and that any new code has been audited and approved by independent parties;
 - Failing securely in the event of errors to avoid compromising the security of the system.
 
-The appraisal process should be auditable and reproducible.
-The integrity of the code and data during execution should be made an explicit objective, for example ensuring that the appraisal functions are computed in an attestable trusted execution environment (TEE).
+It is critical that appraisal procedures are auditable and reproducible.
+The integrity of code and data during execution is an explicit objective, for example, ensuring that the appraisal functions are executed in an attestable trusted execution environment (TEE).
 
 The integrity of public and private key material and the secrecy of private key material must be ensured at all times.
 This includes key material carried in attestation key triples and key material used to verify the authority of triples (such as public keys that identify trusted supply chain actors).
 For more detailed information on protecting Trust Anchors, refer to {{Section 12.4 of -rats-arch}}.
+As it is possible to use the public part of an asymmetric key pair for Evidence generation to be used to identify an Attesting Environment, this method of potentially identifying unique instances of Attesting Environments (and profiling their respective owners) can come with privacy considerations that have to be carefully weighed.
```

Comment:
The text here is not clear to me?

Are we saying:
1. Asymmetric Key Pair used to sign the Evidence OR
2. Public Part of Asymmetric Key pair is part of Evidence Claim used to identify the Attesting Env?

Reply:
Public keys are PII unless they are for group signing (where the size of the group is privacy preserving).

Reply:
> Are we saying:
> 
>     Asymmetric Key Pair used to sign the Evidence OR
>     Public Part of Asymmetric Key pair is part of Evidence Claim used to identify the Attesting Env?

Yes. That happens and is a PII issue.

Reply:
Yes, except DAA needs to be updated to support PQC.